### DISCUSSION TITLE
Evaluating Game Reviews

### DISCUSSION THREAD

**Discussion by OnurAkinn** [quer/the-steam-awards#36] [Ideas]
"Was this comment helpful?" A module that would evaluate "Yes", "No", "Funny" to his question would be nice.

![image](https://user-images.githubusercontent.com/58367885/147407298-e72b1afe-652e-4f46-bda2-becf1b16bd46.png)

**Comment by quer** [comment]
Hey, this shoud have beed added now

https://github.com/quer/the-steam-awards/commit/21c0b65b9d2a651fe027f8b0b7add3802ca66a7f

**Reply by OnurAkinn** [reply]
The module works great. Thanks
